specs-go/config.go

```diff
@@ -430,6 +430,8 @@ type SolarisAnet struct {
 
 // Windows defines the runtime configuration for Windows based containers, including Hyper-V containers.
 type Windows struct {
+	// LayerFolders contains a list of absolute paths to directories containing image layers.
+	LayerFolders []string `json:"layerFolders"`
```

Comment:
> It looks like there are linux fields that use camel case as well.

The Linux config tries to consistently use camelCase.  I don't know if the maintainers care what convention the Windows-only fields use; probably wait for them to weigh in.